Add arcify support.

GoneCaving opened the pull request:
Added Arc support via arcify (https://llllllll.co/t/arcify/22133). Not sure if Lua supports a way of checking for libraries, as it would be nice to be able to check if the user has this without forcing it as a requirement. 

Also added some additional text in the print statements.

GoneCaving committed:
Add arcify support.

oliviercreurer commented:
Amazing -- thanks (and sorry for the slow response)! Got another PR for sample support, so I'll likely bundle the two together in a single release (just waiting on a fix)...

GoneCaving commented:
No problem. I had been meaning to google and see if Lua has a way of checking and handling missing dependencies (so compass would gracefully handle arcify not being installed), but haven’t got around to it. 

D

Sent from my iPhone

> On 28 Sep 2019, at 15:49, Olivier Creurer <notifications@github.com> wrote:
> 
> Amazing -- thanks (and sorry for the slow response)! Got another PR for sample support, so I'll likely bundle the two together in a single release (just waiting on a fix)...
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

justmat commented:
I definitely agree with Tehns approach to this. Just add a ``compass/lib`` folder and put the arcify script inside. That way the version of arcify that compass expects is always available.

GoneCaving committed:
Local copy of arcify as of
  commit: d6b6c98f358735bd048bfc88dcc7064e1c80ea7a
  Author: Nathan Koch <nathanrkoch@gmail.com>
  Date:   Thu Sep 12 10:14:44 2019 -0400

GoneCaving committed:
Reference local copy of arcify

GoneCaving commented:
Added a local copy of arcify as suggested, with readme linking back to the original repo, and commit msg detailing the version.
